mne_bids/write.py

```diff
@@ -608,12 +608,18 @@ def _write_raw_brainvision(raw, bids_fname):
         raise ImportError('pybv >=0.2.0 is required for converting ' +
                           'file to Brainvision format')
     from pybv import write_brainvision
-    events, event_id = events_from_annotations(raw)
+    events, _ = events_from_annotations(raw)
+    events[:, 0] -= raw.first_samp
```

Comment:
can you add a note that it's because brainvision marks events starting from first available data point and ignores the raw.first_samp ?